rook/migration/v1.12/README.md

```diff
@@ -0,0 +1,52 @@
+# Upgrade rook-ceph to v1.12.x
```

Comment:
Doesn't it make sense to make a single migration doc going from the previous set version 1.11 to 1.14? I don't really like that it is split up

Reply:
Looks like v1.11.x-to.v1.14.x have all the steps now. So this and the other readmes could be removed? :slightly_smiling_face: